d2/src/main/java/com/linkedin/d2/xds/XdsToD2PropertiesAdaptor.java

```diff
@@ -219,6 +219,7 @@ public void onChanged(XdsClient.NodeUpdate update)
           }
           catch (PropertySerializationException e)
```

Comment:
line 281 (the cluster data's onChanged method) also needs to add _clusterEventBus.publishInitialize

Reply:
Yeap. Good catch. Updated.